client/src/components/Cards.js

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components'
+
+function Cards({ author, text }) {
+  return (
+    <CardWrapper>
+      <CardText>{text}</CardText>
+      <CardAuthor>{author}</CardAuthor>
+    </CardWrapper>
```

Comment:
Just a way I personally prefer to name components: 
```suggestion
    <Wrapper>
      <Text>{text}</Text>
      <Author>{author}</Author>
    </Wrapper>
```
The reason why is because you are already in your `Card` component. No need to repeat that for every styled component within this component 🙂

Reply:
Done! :-)